### DISCUSSION TITLE
Welcome to v4 Discussions!

### DISCUSSION THREAD

**Discussion by qerionx** [xorynix/dogeub#115] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by porgman102** [comment]
any update on the local deploy error?

**Reply by ab36211** [reply]
I don't think the dev team found a solution yet.

**Reply by TurtleTheCoder** [reply]
it still doesn't work :(

**Comment by antplup** [comment]
can someone tell me the port id

**Reply by deltongraves-png** [reply]
how do you get on the website
